src/CameraControls.ts

```diff
@@ -159,6 +157,11 @@ export class CameraControls extends EventDispatcher {
 	protected _needsUpdate = true;
 	protected _updatedLastTime = false;
 
+	private _dragStartPosition: _THREE.Vector2;
+	private _lastDragPosition: _THREE.Vector2;
+	private _dollyStart: _THREE.Vector2;
+	private _elementRect: _THREE.Vector4;
```

Comment:
er, sorry I meant,
`protected` for `_elementRect`
and revert `_dragStartPosition`, `_lastDragPosition` and `_dollyStart` as scoped

Reply:
@yomotsu actually we cant have `dragStartPosition`, `_lastDragPosition` and `_dollyStart` as scoped. They are used my multiple class functions. We can have then as real private with `#` if you want